samples/net/dhcpv4_client/src/main.c

```diff
@@ -46,20 +52,20 @@ static void handler(struct net_mgmt_event_callback *cb,
 			continue;
 		}
 
-		LOG_INF("Your address: %s",
+		LOG_INF("   Address[%s]: %s", net_if_get_device(iface)->name,
```

Comment:
IMHO the device name is not very useful for networks, better would be to print the interface index as that is used in many places in the network stack. If needed, one can then see the index <-> device name mapping in network shell using "net iface" command.

Reply:
Thank you for reviewing. Fixed it.
I changed to show index <-> device name mapping at dhcp client starting.